src/SelfServe/SelfServeUtils.tsx

```diff
@@ -175,13 +191,9 @@ const getInput = (value: DecoratorProperties): AnyDisplay => {
   }
 };
 
-export const getMessageBarType = (type: SelfServeNotificationType): MessageBarType => {
-  switch (type) {
-    case SelfServeNotificationType.info:
-      return MessageBarType.info;
-    case SelfServeNotificationType.warning:
-      return MessageBarType.warning;
-    case SelfServeNotificationType.error:
-      return MessageBarType.error;
-  }
+export const generateBladeLink = (blade: BladeType): string => {
+  const subscriptionId = userContext.subscriptionId;
+  const resourceGroupName = userContext.resourceGroup;
+  const databaseAccountName = userContext.databaseAccount.name;
+  return `www.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/${subscriptionId}/resourceGroups/${resourceGroupName}/providers/Microsoft.DocumentDb/databaseAccounts/${databaseAccountName}/${blade}`;
```

Comment:
>`www.portal.azure.com [](start = 9, length = 21)

use a helper method so that right url can be used depending on the environment

Reply:
will do in next PR after adding portal side support for passing this param